case_mapping/uco/observable.py

```diff
@@ -702,23 +708,21 @@ def __init__(
                 "uco-observable:application": application,
                 "uco-observable:from": call_from,
                 "uco-observable:to": call_to,
+                "uco-observable:participant": call_participant,
             }
         )
 
 
 class FacetPhoneAccount(FacetEntity):
-    def __init__(self, phone_number=None, display_name=None):
```

Comment:
Good catch. This change is consistent with the ontology and the CASE examples (CASE-Examples and website).